guides/doc-Administering_Red_Hat_Satellite/topics/proc_estimating-the-size-of-a-backup.adoc

```diff
@@ -20,18 +20,18 @@ A full backup requires space to store the following data:
 . Enter the `du` command to estimate the size of uncompressed directories containing {Project} database and configuration files:
 +
 ----
-# du -sh /var/lib/pgsql/data /var/lib/pulp
-100G    /var/lib/pgsql/data
+# du -sh /var/opt/rh/rh-postgresql12/lib/pgsql/data /var/lib/pulp
+100G  /var/opt/rh/rh-postgresql12/lib/pgsql/data
```

Comment:
Also EL7 vs EL8 here.

Reply:
EL8 updates will be needed in multiple places. My intent is to tackle those as separate across the board update rather than conflate it with scrubbing the current guide of Pulp 2 and replacing with Pulp 3.